scripts/gfxrecon-replay-renamed.py

```diff
@@ -58,6 +58,7 @@ def run_command(command):
                                 stdout=subprocess.PIPE,
                                 stderr=subprocess.PIPE,
                                 universal_newlines=True)
+    stdout, stderr = process.communicate()
```

Comment:
This gets any available output from `process` and then discards it because no code uses the resulting variables "stdout" and "stderr".  (https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate - `Read data from stdout and stderr, until end-of-file is reached. Wait for process to terminate`)  I think the loop afterward is supposed to drain output from the subprocess but after this call to `communicate` will have no results.  This PR should probably print the `stdout` and `stderr` returned and then should probably delete the `poll` loop afterwards.